pylint/checkers/__init__.py

```diff
@@ -39,7 +39,8 @@
 15: stdlib
 16: python3
 17: refactoring
-18-50: not yet used: reserved for future internal checkers.
+18: threading
```

Comment:
I couldn't find where those numbers are defined

Reply:
Could probably be handled in #5248 along with the consistency fixes what do you think @DanielNoord ?

Reply:
I would think #5255 might be better for this. We could base the test on an update list of checkers and their message id prefixes? #5248 was more meant as an easy script to point (new) contributors to when making new checkers.